docs/index.md

```diff
@@ -98,6 +98,7 @@ update time.
 | zfs_data_compression     | String                        | Specifies a compression algorithm used for the VM's data dataset                                                                                                                                                          | No                  | Yes    | Yes    |
 | zfs_io_priority          | Number                        | ZFS IO Priority                                                                                                                                                                                                           | Yes                 | Yes    | Yes    |
 | zlog_max_size            | Number                        | Sets the maximum size of the stdio.log file for a docker zone before rotation. NOTE: To be used by sdc-docker only.                                                                                                       | No                  | Yes    | Yes    |
+| zfs_snapshot_limit       | Number                        | Sets the number of ZFS snapshots a VM can take.                                                                                            | Yes                 | Yes    | Yes    |
```

Comment:
move this up a line to be in alphabetical order, please

Reply:
Damn, off-by-one.